templates/themes/modern/search_article.html

```
@@ -28,22 +28,22 @@
             <h3 class="text-xl font-bold text-on-surface-variant">没有搜索到相关专栏</h3>
             <p class="text-on-surface-variant/70 mt-2">换个关键词试试？</p>
         </div>
     {% else %}
         <div class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-6">
             {% for ar in rs %}
-                <div class="group bg-surface dark:bg-surface-variant/20 rounded-2xl overflow-hidden hover:bg-surface-variant/30 flex flex-col transition-all duration-300 hover:shadow-lg hover:-translate-y-1">
+                <div class="group bg-surface dark:bg-surface-variant/20 rounded-3xl overflow-hidden hover:bg-surface-variant/30 flex flex-col transition-all duration-300 hover:shadow-lg hover:-translate-y-1">
                     <!-- Thumbnail -->
                     <a href="/read/cv{{ ar['id'] }}"
                        class="relative block aspect-video overflow-hidden">
                         <img loading="lazy"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)